packages/prosess-beregningsgrunnlag/src/BeregningsgrunnlagProsessIndex.tsx

```diff
@@ -117,6 +131,12 @@ const BeregningsgrunnlagProsessIndex: FunctionComponent<
   formData,
   setFormData,
 }) => {
+  if (
+    beregningsgrunnlagListe.length === 0 &&
+    beregningsgrunnlagsvilkar?.perioder?.some(periode => periode.avslagKode === '1043')
+  ) {
+    return ikkeTilstrekkeligInntektsgrunnlag();
```

Comment:
Kva med å heller berre ta inn beregningsgrunnlagsvilkar som parameter til visningForManglendeBG og så sjekke om det finnes avslagsårsak der? Stylinga av visningen skal vel vere lik for dei to.